action.yml

```diff
@@ -1,8 +1,14 @@
-name: 'Wait'
+name: 'Constable'
 description: 'Create a JavaScript Action grade the project'
+inputs:
```

Comment:
Just remove this. I think its throwing an error in the test GH action workflow

Reply:
Done